packages/formation/sass/modules/_m-button.scss

```diff
@@ -3,33 +3,41 @@
 .usa-button:visited,
 .usa-button-primary:visited,
 button,
-[type="button"],
-[type="submit"],
-[type="reset"],
-[type="image"] {
+[type='button'],
+[type='submit'],
+[type='reset'],
+[type='image'] {
   background-color: $color-primary;
 
   &.usa-button-secondary {
     background: transparent;
+    &:focus,
+    &:hover {
+      background-color: $color-cool-blue-lightest;
+    }
   }
 }
 
+.usa-button:focus {
+  background: $color-primary-darker;
+}
```

Comment:
Should this be on `usa-button-primary` instead?

Reply:
This effects the Primary Button section which is `.usa-button`
![image](https://user-images.githubusercontent.com/72466113/135665719-a165b758-eb17-4b6d-87f5-d4301a0a9c84.png)